src/components/MovieView.js

```diff
@@ -1,20 +1,52 @@
 import React from 'react';
+import { Component } from 'react/cjs/react.production.min';
```

Comment:
This is cool. I'm unfamiliar with this syntax! I think though that we should change this files import syntax to mirror how we've imported the component class for App.js